superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts

```diff
@@ -77,12 +77,17 @@ describe('AdhocFilters', () => {
 
     cy.wait('@filterValues');
 
-    cy.get('#filter-edit-popover #adhoc-filter-edit-tabs-tab-SQL').click();
-    cy.get('#filter-edit-popover .ace_content').click();
-    cy.get('#filter-edit-popover .ace_text-input').type(
-      "'Amy' OR name = 'Bob'",
-    );
-    cy.get('#filter-edit-popover button').contains('Save').click();
+    cy.get('[data-test="filter-edit-popover"]')
+      .find('[data-test="adhoc-filter-edit-tabs"]')
+      .contains('Custom SQL')
```

Comment:
Curious if you think we need to do the get->find->contains->click work here, or if we should just make an explicit/specific attribute here and use it directly, e.g. `cy.get('[data-test="adhoc-filter-edit-tab-custom-sql"]').click()